Makefile

```diff
@@ -2,7 +2,7 @@ init:
 	pip install -r requirements.txt
 
 TEST_ARGS = --doctest-modules --doctest-continue-on-failure --cov matchzoo/ --cov-report term-missing --cov-report html --cov-config .coveragerc matchzoo/ tests/ -W ignore::DeprecationWarning
-FLAKE_ARGS = ./matchzoo --exclude=__init__.py,matchzoo/contrib
+FLAKE_ARGS = ./matchzoo --exclude=__init__.py,matchzoo/contrib/models
```

Comment:
why

Reply:
somehow the author wants to flake8 other contrib stuff?

Reply:
I misunderstood it and removed the changes to Makefile.